runners/direct-java/src/test/java/org/apache/beam/runners/direct/portable/FlattenEvaluatorFactoryTest.java

```diff
@@ -49,30 +47,44 @@
 
   @Rule public TestPipeline p = TestPipeline.create().enableAbandonedNodeEnforcement(false);
 
+  @Ignore("Enable when the Flatten Evaluator Factory is fully migrated")
   @Test
   public void testFlattenInMemoryEvaluator() throws Exception {
-    PCollection<Integer> left = p.apply("left", Create.of(1, 2, 4));
-    PCollection<Integer> right = p.apply("right", Create.of(-1, 2, -4));
-    PCollectionList<Integer> list = PCollectionList.of(left).and(right);
-
-    PCollection<Integer> flattened = list.apply(Flatten.pCollections());
-
-    CommittedBundle<Integer> leftBundle = bundleFactory.createBundle(left).commit(Instant.now());
-    CommittedBundle<Integer> rightBundle = bundleFactory.createBundle(right).commit(Instant.now());
+    PCollectionNode left =
```

Comment:
Use ExecutableGraphBuilder?

Reply:
It doesn't work in this context, because I haven't added joins (and I'd probably like to avoid to for the time being)